File: Lib/test.nexss/nexssLog.test.js
const {
  nxsInfo,
  nxsOk,
  nxsWarn,
  nxsError
} = require(`${process.env.NEXSS_PACKAGES_PATH}/Nexss/Lib/NexssLog.js`);

const NexssIn = require(`${process.env.NEXSS_PACKAGES_PATH}/Nexss/Lib/NexssIn.js`);

// console.log(JSON.stringify());

let NexssStdout = NexssIn();

nxsInfo("Nexss info message");
nxsOk("Nexss ok message");
nxsWarn("Nexss warn message");
nxsError("Nexss error message");

delete NexssStdout.resultField_1;
delete NexssStdout.nxsIn;
process.stdout.write(JSON.stringify(NexssStdout));

File: Lib/test.nexss/nexssLog.test.py
# Nexss PROGRAMMER - Blender/Python3
# Python 3.7
import json
import sys
import os
sys.path.append(os.getenv("NEXSS_PACKAGES_PATH") + "\\Nexss\\Lib\\")
from NexssIn import *

nxsInfo("Nexss info message")
nxsOk("Nexss ok message")
nxsWarn("Nexss warn message")
nxsError("Nexss error message")

NexssStdout = json.dumps(NexssStdout, ensure_ascii=False).encode('utf8','surrogateescape')
# STDOUT
print(NexssStdout.decode('utf8','surrogateescape'))
